File: Lesson5/Browser/app/src/main/java/com/example/browser/MainActivity.java
package com.example.browser;

import android.content.Intent;
import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

import java.net.URI;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void browser(View view) {

        Intent intent = new Intent(getBaseContext(), BrowserActivity.class);

        EditText input = findViewById(R.id.etInput);

        String url = input.getText().toString();

        intent.setData(Uri.parse(url));

        startActivity(intent);
    }
}

File: Lesson19/Location/app/src/main/java/com/example/location/MainActivity.kt
package com.example.location

import android.content.Context
import android.location.LocationManager
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val locationManager = getSystemService(Context.LOCATION_SERVICE) as LocationManager

        val provider = locationManager.allProviders

        provider.forEach {
            Log.e("PROVIDER", it)
        }

        Log.e("IS_ENABLED", locationManager.isLocationEnabled.toString())
    }
}
